Makefile

```diff
@@ -2,7 +2,7 @@ default:
 	@echo "an explicit target is required"
 
 # easier to test python2 vs. python3
-PYTHON=python3
+PYTHON=pypy3
```

Comment:
Can you comment on the benefits of this change?

Reply:
We were previously using pypy for tests as it runs the entire test suite in ~10ish minutes on my laptop as compared to ~20+ for regular python. A large number of the tests involve our graph matching algorithm, which while designed to be efficient, is solving an NP-Complete problem of sub-graph isomorphism matching. I'd run into problems getting the local install of pypy working so I had changed this back to python3 so I could still test even if slower but with the update to 3.7 I got pypy working again locally.